vrecord

```diff
@@ -1139,10 +1163,10 @@ drawgrid=w=256:h=256:t=1:c=white@0.2"
 # list of selections for each vrecord option
 VIDEO_INPUT_OPTIONS=("Composite" "SDI" "Component" "S-Video")
 AUDIO_INPUT_OPTIONS=("Analog" "SDI Embedded Audio" "Digital Audio (AES/EBU)")
-CONTAINER_OPTIONS=("QuickTime" "Matroska" "AVI" "MXF")
-VIDEO_CODEC_OPTIONS=("Uncompressed Video" "FFV1 version 3" "JPEG2000" "ProRes" )
+CONTAINER_OPTIONS=("QuickTime" "Matroska" "AVI" "MXF" "MP4")
+VIDEO_CODEC_OPTIONS=("Uncompressed Video" "FFV1 version 3" "JPEG2000" "ProRes" "ProRes (HQ)" "h264")
```

Comment:
Sorry for nit picking: should we not say `H.264` rather than `h264`? (Also at line 844.)